### DISCUSSION TITLE
Open ticket and set status: "Response received" - fails

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16703] [General]
"Response received" is not changing ticket status.
Email is attached to ticket correctly
I checked in 2 own fresh 6.4 instances. Cannot verify in dev test env, so if possible can anyone confirm it works or not?
One remark... Helpdesk has been converted to advanced module. Inventory fields are NOT mandatory.

**Comment by OkayashiM** [comment]
Checked your scenario without converting to advanced module. Status not changing.
Related files seems to be ok: OSSMailScanner.php and BindHelpDesk.php

**Comment by s2b-git** [comment]
Logs here shows:
![image](https://user-images.githubusercontent.com/84110093/205417575-cd09902a-3c78-4db8-a34b-359337aa63a3.png)

It seems like record has been deleted, and this causing lock. Question is how to reset it :/

**Reply by s2b-git** [reply]
It comes to the point where Mail Scanner Bind never ends its job

![image](https://user-images.githubusercontent.com/84110093/205421149-7c1f730e-e7d7-45f4-b34c-f98ae7128d12.png)

**Reply by TonyM15** [reply]
U have deleted record directly in database I think.  
Not sure how to fix it but people will help here. Would be good to know HowTo.

Happy Sunday!

**Reply by s2b-git** [reply]
Fixed it by removing orphaned id's from s_yf_mail_relation_updater, but tickets status still not updating. Logs showing absolutely nothing :o)

**Comment by JohnMaverick** [comment]
In the past been playing around ticketing too, but found it a bit too complicated. Remember that MailscannerBind was returning errors what was hard to debug. In my opinion workflows including mailscanner (as they often hyper essential for business) should have some option to debug in line, and/or throw error log but carry on work. Currently if mailscanner hang on some unexpected thing, it becoming a frustrating if admin not sitting in front of his console.

**Comment by ScottAndaman** [comment]
Mailscanner not changing ticket status here too.

Scott

**Comment by garryatbrighton** [comment]
6.4 - dont see it works
What worry me no other users confirm or deny this issue. Maybe weekend it the case

**Reply by ScottAndaman** [reply]
I noticed that when you are using one mailbox in mailscanner (with multiple identies) it works. When you are using multiple email accounts in mailscanner status is not changing, This is bug.
Mailscanner needs really attention.

**Reply by garryatbrighton** [reply]
Incredibly difficult to get how stuff works like in this case. Maybe developer can throw some clue how to organize it so it dealing with multiple mailboxes and updating statuses?
